examples/trait-erc20/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "erc20"
```

Comment:
Should we change the name of trait-ified examples to `name = "trait-erc20"`? The `trait-flipper` crate is currently also still named `flipper`.

Reply:
I think that is fine.